docs/source/pillars/computing_technology.md

```diff
@@ -419,3 +419,236 @@ This {term}`business process <business process>` involves the {ref}`TRE operator
+    If security flaws are identified in a test, it may not be sensible to publicise these until a fix is in place.
+  - Recommended
+```
+
+(encryption)=
+
+### Encryption
+
+The ability of the {ref}`TRE operator <infrastructure_roles>` to deploy and manage encryption to protect information assets, including data for TRE research projects.
+
+Here we define 'project' data as the data brought in for work which is very likely to be sensitive and 'user' data, as the working files of a project which might hold copies of all or part of the project data or otherwise reveal sensitive data (_e.g._ through hard coded row/column names).
+
+```{list-table}
+:header-rows: 1
+:name: tab-computing-encryption
+
+* -
+  - Statement
+  - Guidance
+  - Importance
+* - 2.5.12.
+  - Your TRE must encrypt project and user data at rest.
+  - This prevents unauthorised access to the data even if the storage media is compromised.
+    This may involve encrypted filesystems or tools to encrypt and decrypt data on demand.
+    The encryption keys may be managed by the {ref}`TRE operator <infrastructure_roles>` or by a trusted external actor, for example a cloud provider.
+  - Mandatory
+* - 2.5.13.
+  - Your TRE must encrypt data when in transit between the TRE and external networks or computers.
+  - Data encryption must be used to safeguard against interception or tampering during transmission.
+    This includes both data ingress and egress and users accessing the TRE, for example over a remote desktop or shell session.
+  - Mandatory
+* - 2.5.14.
+  - Your TRE should encrypt data when in transit inside the TRE.
+  - If possible, data transfers between different components of a TRE should also be encrypted.
+  - Recommended
+* - 2.5.15.
+  - You should use encryption algorithms and software that are widely accepted as secure.
+  - Encryption algorithms widely accepted as secure today may become insecure in the future, for instance due to newly-identified flaws, or advances in compute capabilities.
+    The latest security patches and updates should be applied to any encryption software being used by the TRE.
+    This helps address any known vulnerabilities or weaknesses in the encryption implementation.
+  - Recommended
+```
+
+### Key Management Application
+
+Software or tools dedicated to generating, storing, and managing encryption keys securely, ensuring their availability and protection.
+
+```{list-table}
+:header-rows: 1
+:name: tab-computing-keymanagement
+
+* -
+  - Statement
+  - Guidance
+  - Importance
+* - 2.5.16.
+  - Your TRE should use secure key management.
+  - TREs should employ secure key management practices, including storing encryption keys separately from the encrypted data and implementing strong access controls (_e.g._ Single Sign On) for key management systems.
+  - Recommended
+```
+
+(physical-security)=
+
+### Physical security
```

Comment:
@machintim recommends moving this to Supporting Capabilities, but I've left it here for now.